src/fluree/db/branch.cljc

```diff
@@ -192,11 +201,12 @@
   then the ledger's latest commit t (in branch-data). The db 't' and db commit 't'
   should be the same at this point (just after committing the db). The ledger's latest
   't' should be the same (if just updating an index) or after the db's 't' value."
-  [{:keys [state index-queue] :as branch-map} new-db index-files-ch]
+  [{:keys [state index-queue indexing-opts] :as branch-map} new-db index-files-ch]
   (let [updated-db (-> state
                        (swap! update-commit (policy/root-db new-db))
                        :current-db)]
-    (enqueue-index! index-queue updated-db index-files-ch)
+    (when-not (:indexing-disabled indexing-opts)
```

Comment:
What sets the `:indexing-disabled` option?

Reply:
@zonotope this is set with the connection config the same way other indexing options are set (like min/max reindex values).